lib/presentation/widget/shared_space/file_surfing/workgroup_nodes_surfing_viewmodel.dart

```diff
@@ -131,7 +131,7 @@ class WorkGroupNodesSurfingViewModel extends BaseViewModel {
               loadAllChildNodes();
             } else if (success is CreateSharedSpaceFolderViewState) {
               loadAllChildNodes();
-            } else if (success is SharedSpaceViewState) {
+            } else if (success is SharedSpaceViewState || success is SharedSpacesViewState) {
```

Comment:
oh we have both `SharedSpaceViewState` and `SharedSpacesViewState` in our project?

Reply:
I think this is confused. Can we merge it to one view state only ?

Reply:
Made it because `SharedSpacesViewState` contains a list and `SharedSpaceViewState` contains one shared space